docs/guide/apm-data-model.asciidoc

```diff
@@ -144,4 +157,100 @@ For a full list of tracked metrics, see the relevant agent documentation:
 * {apm-java-ref-v}/metrics.html[Java]
 * {apm-node-ref-v}/metrics.html[Node.js]
 * {apm-py-ref-v}/metrics.html[Python]
-* {apm-ruby-ref-v}/metrics.html[Ruby]
\ No newline at end of file
+* {apm-ruby-ref-v}/metrics.html[Ruby]
+
+// This heading is linked to from the APM UI section in Kibana
+[[metadata]]
+=== Metadata
+
+Metadata can enrich your events and make application performance monitoring even more useful.
+Let's explore the different types of metadata that Elastic APM offers.
+
+[float]
+[[labels-fields]]
+==== Labels
+
+Labels are used to add *indexed* information to transactions, spans, and errors.
+Indexed means the data is searchable and aggregatable in Elasticsearch.
+Multiple labels can be defined with different key-value pairs.
+
+* Indexed: Yes
+* Elasticsearch type: {ref}/object.html[object]
+* Elasticsearch field: `labels` (previously `context.tags` in <v.7.0)
+* Applies to <<transactions>> | <<transaction-spans>> | <<errors>>
+
+Label values can be a string, boolean, or number.
+Note that some agents only support string values at this time.
+Because labels are stored in the same place in Elasticsearch, each value must have the same data type.
```

Comment:
```suggestion
Because labels for a given key are stored in the same place in Elasticsearch, all label values of a given key must have the same data type.
```

In addition, I am not sure what "stored in the same place in Elasticsearch" means.

Reply:
Good call on this @eyalkoren, thanks! "Stored in the same place" is confusing. It's meant that the key "foo" from multiple agents is stored in the same place in ES. I'll make this more clear